python/grass/pygrass/utils.py

```diff
@@ -5,13 +5,16 @@
 
 import grass.lib.gis as libgis
 
+# flake8: noqa: E402
 libgis.G_gisinit("")
 
 import grass.lib.raster as libraster
 from grass.lib.ctypes_preamble import String
 from grass.script import core as grasscore
 from grass.script import utils as grassutils
```

Comment:
If moving these two lines out of the flake8 exclusion (below), does isort give the same sorting?

Reply:
I have moved both of them out of flake8, isort didnt change anything